components/piu.rst

```diff
@@ -38,3 +38,10 @@ How to configure
 .. code-block:: bash
 
     $ cat ~/Library/Application\ Support/piu/piu.yaml
+
+An example configuration:
+
+.. code-block:: json
```

Comment:
haha.  Now I am wondering why I was in an undefined state - a yaml file with a json structure inside?